db/migrate/20150922165007_create_ex_links.rb

```diff
@@ -0,0 +1,15 @@
+class CreateExLinks < ActiveRecord::Migration
+  def change
+    create_table :ex_links do |t|
+      t.string :label
+      t.text :url
+      t.string :tags
```

Comment:
What I mean is this:

```
table exlinks(integer id, ...)
table exlink_tags(integer id, integer exlink_id, varchar tag...)
```

Serialisation can be an option, but mostly as a fallback for data that is really store-only, or the logic is very complicated. If you fall back to serialisation, you are throwing away all the leverage that your database gives you. The database is really good at searching and processing large amounts of data.